src/test/java/kitchenpos/application/MenuGroupServiceTest.java

```diff
@@ -0,0 +1,77 @@
+package kitchenpos.application;
+
+import kitchenpos.dao.MenuGroupDao;
+import kitchenpos.domain.MenuGroup;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Mockito.when;
+
+@ExtendWith(MockitoExtension.class)
+class MenuGroupServiceTest {
+
+    private MenuGroup menuGroup1;
+    private MenuGroup menuGroup2;
+    private MenuGroup menuGroup3;
+
+    private List<MenuGroup> menuGroups = new ArrayList<>();
+    private MenuGroupService menuGroupService;
+
+    @Mock
+    private MenuGroupDao menuGroupDao;
+
+    @BeforeEach
+    public void setUp() {
+        menuGroup1 = new MenuGroup("치킨");
```

Comment:
```suggestion
        치킨 = new MenuGroup("치킨");
```
테스트 코드도 계속 유지보수하고 관리해야되는 코드입니다
불용어보다 의미 있는 변수명을 추천합니다